okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -65,6 +66,16 @@ protected RealCall(OkHttpClient client, Request originalRequest) {
     }
   }
 
+  private void captureCallStackTrace() {
+    StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
```

Comment:
One other option that’s tempting . . . what if we asked `java.util.logging` whether the logger for say, `okhttp3.OkHttpClient` was enabled at `Level.INFO` ?